### DISCUSSION TITLE
How to enable per pixel scrolling like neovide in ghostty

### DISCUSSION THREAD

**Discussion by rakshith-git** [ghostty-org/ghostty#3374] [Q&A] [answered]
In neovide terminal emulator you have the ability to get smooth scrolling 
and animations wherein you can scroll on a pixel level , i think the only other terminal that does this is sakura terminal . It does seem like ghostty can already do this too as its gpu rendered ,just dont know how to enable it 
![SmoothScrolling (1)](https://github.com/user-attachments/assets/a3f508d0-d76d-46b4-9890-b88af6d1d7f1)

**Comment by JakeWharton** [comment, marked as answer]
https://github.com/ghostty-org/ghostty/discussions/3206
